src/OpenTelemetry.Exporter.Zipkin/.publicApi/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,6 @@
+OpenTelemetry.Exporter.ZipkinExporterOptions.BatchExportProcessorOptions.get -> OpenTelemetry.BatchExportProcessorOptions<System.Diagnostics.Activity!>!
```

Comment:
This seems incorrect. We shouldn't be moving shipped APIs to unshipped. You can probably just move these back into the shipped file.